region/brinstar/green/Early Supers Room.json

```diff
@@ -586,6 +586,43 @@
       "requires": [],
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 4],
+      "name": "G-Mode Overload PLMS with Camera Scroll Blocks",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": false
+        }
+      },
+      "requires": [
+        {"or": [
+          "canWalljump",
+          {"and": [
```

Comment:
Hijump + speed
shinespark... you can run on the crumbles
HiJump springball
speed springball

Reply:
But then you don't need GMode.